lib/ansible/cli/doc.py

```diff
@@ -332,10 +332,11 @@ def tty_ify(cls, text):
         t = cls._CONST.sub("`" + r"\1" + "'", t)        # C(word) => `word'
         t = cls._RULER.sub("\n{0}\n".format("-" * 13), t)   # HORIZONTALLINE => -------
 
-        t = cls._REFTAG.sub(r"", t)  # remove rst :ref:
-        t = cls._TERM.sub(r"", t)  # remove rst :term:
-        t = cls._NOTES.sub(r" Note:", t)  # nicer note
-        t = cls._SEEALSO.sub(r"", t)  # remove seealso
+        # remove rst
+        t = cls._RST_SEEALSO.sub(r"See website for:", t) # seealso is special and need to break
+        t = cls._RST_NOTE.sub(r"note:", t)               # .. note:: to note:
```

Comment:
I thought we decided on titlecase, no?
```suggestion
        t = cls._RST_NOTE.sub(r"Note:", t)               # .. note:: to note:
```